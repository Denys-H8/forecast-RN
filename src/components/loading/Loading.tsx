import React from 'react';
import { View, ActivityIndicator } from 'react-native';

import { styles } from './loading.styles';
import constants from 'src/constants';

const Loading: React.FC = () => (
  <View style={styles.container}>
    <ActivityIndicator size="large" color={constants.MAIN_COLOR} />
  </View>
);

export default Loading;
