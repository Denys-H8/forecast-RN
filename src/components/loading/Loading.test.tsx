import React from 'react';
import { render } from '@testing-library/react-native';
import Loading from './Loading';

test('Loading snapshot', () => {
  const { toJSON } = render(<Loading />);
  expect(toJSON()).toMatchSnapshot();
});
