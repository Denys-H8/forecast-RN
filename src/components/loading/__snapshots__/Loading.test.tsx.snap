// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#191C39",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <ActivityIndicator
    animating={true}
    color="#fff"
    hidesWhenStopped={true}
    size="large"
  />
</View>
`;
